src/components/StarterCard.tsx

```diff
@@ -0,0 +1,68 @@
+import { Pokemon } from "@/types";
+import { Stat, statToDisplayString } from "@/utils/stat";
+import React from "react";
+import { Card, CardContent, CardHeader, CardTitle } from "./ui/card";
+import { cn } from "./lib/utils";
+import IvChart from "./IvChart";
+
+type StarterCardProps = React.HTMLAttributes<HTMLDivElement> & {
+  pokemon: Pokemon;
+};
+
+const StarterCard: React.FC<StarterCardProps> = ({ pokemon, className }) => {
+  const statIncreased = pokemon.nature.increased as Stat;
+  const statDecreased = pokemon.nature.decreased as Stat;
+
+  return (
+    <Card className="w-full max-w-3xl relative">
+      <CardHeader className="pb-4">
+        <CardTitle className="text-2xl font-bold">
+          <div className="flex items-center">
+            <div>{pokemon.name}</div>
+            <div
+              className={cn(
+                "w-8 h-8 ml-2 rounded-full flex items-center justify-center",
+                pokemon.gender === "male" ? "bg-blue-500" : "bg-pink-500",
```

Comment:
Can we use the PokemonGender enum here and below this?

Reply:
Oh yeah absolutely. 

I think that is also just broken as well (because "male" instead of "Male")

From what I remember I just copy pasted PokemonCard and tweaked it from there
I guess I worked on it before your fix was merged.